src/set_theory/surreal/basic.lean

```diff
@@ -96,6 +96,34 @@ theorem lt_of_lf {x y : pgame} (ox : numeric x) (oy : numeric y) (h : x ⧏ y) :
 theorem lf_iff_lt {x y : pgame} (ox : numeric x) (oy : numeric y) : x ⧏ y ↔ x < y :=
 ⟨lt_of_lf ox oy, lf_of_lt⟩
 
+/-- Definition of `x ≤ y` on numeric pre-games, in terms of `<` -/
+theorem le_def_lt {x y : pgame} (ox : x.numeric) (oy : y.numeric) :
```

Comment:
Maybe `le_iff_forall_lt` would be a better name?

Reply:
We already have `le_def_lf` so this name is much more consistent.

Reply:
Let's just change `le_def_lf` too. It's brand new anyway, so not too late to improve the naming. :-)

Reply:
I've renamed the lemmas and added `of_` variants. The more general rename you suggest has been made into PR #14206.